components/forms/Form/Form.tsx

```diff
@@ -45,6 +49,36 @@ const InnerForm = (props: InnerFormProps & FormikProps<FormValues> & DynamicForm
     }
   }, [formStatusError, errorList, lastSubmitCount, canFocusOnError]);
 
+  useEffect(() => {
+    if (!formConfig) {
+      return;
+    }
+
+    // calculate initial delay for submit timer
+    const secondsBaseDelay = 2;
+    const secondsPerFormElement = 2;
+    const numberOfRequiredElements = formConfig.elements.filter(
```

Comment:
I refactored a bit because `(element)` will never be 'undefined'

Reply:
That is correct! @jeberhardt  I saw you added this as part of a "Testing fix" commit so I thought you would end up removing it 😆

Reply:
Yes, I added it, because I thought it might have had something to do with why the Cypress tests were failing (since that's the part of the code that I changed). As it turns out, the Cypress tests failing had nothing to do with that, and the null check on `elements` was totally not needed.